controller/backup_volume_controller.go

```diff
@@ -283,8 +283,10 @@ func (bvc *BackupVolumeController) reconcile(backupVolumeName string) (err error
 	clustersSet := sets.NewString()
 	for _, b := range clusterBackups {
 		// Skip the Backup CR which is created from the local cluster and
-		// the snapshot backup hasn't be completed or pulled from the remote backup target yet
-		if b.Spec.SnapshotName != "" && b.Status.State != types.BackupStateCompleted {
+		// the snapshot backup hasn't be completed and sync with the remote backup target yet
+		// note that the error/unknown Backup CRs would be included in the clustersSet
+		if b.Status.LastSyncedAt.IsZero() && b.Spec.SnapshotName != "" &&
+			(b.Status.State == types.BackupStateNew || b.Status.State == types.BackupStateInProgress || b.Status.State == types.BackupStateCompleted) {
```

Comment:
Can the state be completed while `LastSyncedAt` is empty?
And it's better to use a separate function like `IsBackupInProgress` to abstract this part.